common/os/mac.cc

```diff
@@ -58,9 +56,8 @@ bool amIBeingDebugged()
 
     // Call sysctl.
 
-    size = sizeof(info);
-    junk = sysctl(mib, sizeof(mib) / sizeof(*mib), &info, &size, nullptr, 0);
-    assert(junk == 0);
+    size_t size = sizeof(info);
+    assert(sysctl(mib, sizeof(mib) / sizeof(*mib), &info, &size, nullptr, 0) == 0);
```

Comment:
Ahhh...realized this is not correct in the general case, because when assertions are disabled (e.g. in an opt build), the `sysctl` doesn't get executed.  I think this needs to be solved a different way.